src/AbstractSQLCompiler.ts

```diff
@@ -213,9 +222,11 @@ export type ValuesNodeTypes =
 	| NullNode
 	| BindNode
 	| TextNode
+	| JSONNode
 	| NumberNode;
 
 export type AliasNode<T> = ['Alias', T, string];
+export type JSONPathNode = ['JSONPath', string];
```

Comment:
Could possibly handle `string[]` instead of `string` and `join()` during SQL generation. Feels like it could be nicer, but probably wouldn't be portable if/when `ExtractJSONPathAsText` is extended to work with MySQL/SQLite.

Reply:
It should be a `string[]` based on the postgres docs at least: https://www.postgresql.org/docs/14/functions-json.html#FUNCTIONS-JSON-OP-TABLE
```
json #>> text[] → text
jsonb #>> text[] → text
```